index.html

```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
                     <div class="bulb"></div>
                 </div>
             </div>
+        </section>
+    <section class="section-4">
+        <div class="mobile-form-wrapper">
+            <h3 class="get-started">Get Started</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name" required></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"required></label>
+            <label for="Message"><textarea type="text" name="" maxlength="500" class="message" placeholder="Enter your message here.." required></textarea></label>
```

Comment:
As I mentioned earlier, please fix input left side padding like on the Figma template. It needs to be 16px.
<img width="499" alt="Screenshot 2023-01-19 at 11 18 02 PM" src="https://user-images.githubusercontent.com/25887253/213551620-ad7fa178-656a-436c-9c9e-24f175cc4687.png">
<img width="554" alt="Screenshot 2023-01-19 at 11 19 02 PM" src="https://user-images.githubusercontent.com/25887253/213551623-04752acf-57ae-40d7-90fd-c593fc9c7ec0.png">

Reply:
@huseyinbiyik hello again do i have to implement the input font as well shown in the figma file?

Reply:
Yes, yours is 13.33px, the template has 14px. You can fix it. 👍

Reply:
@huseyinbiyik please could you suggest how to do this ? i know its not a change you suggested but id like to implement it for knowledge purposes